code-studio/src/js/components/progress/course_progress.jsx

```diff
@@ -24,11 +25,25 @@ const CourseProgress = React.createClass({
   propTypes: {
     professionalLearningCourse: React.PropTypes.bool,
     focusAreaPositions: React.PropTypes.arrayOf(React.PropTypes.number),
-    stages: React.PropTypes.arrayOf(stageShape)
+    stages: React.PropTypes.arrayOf(stageShape),
+    peerReviewsRequired: React.PropTypes.number,
+    peerReviewsPerformed: React.PropTypes.arrayOf(peerReviewShape)
   },
 
   render() {
     const groups = _.groupBy(this.props.stages, stage => (stage.flex_category || 'Content'));
+    const peerReviewSection = true && (
```

Comment:
I'm working on making this only show up for scripts where the user has a valid plc enrollment.